### DISCUSSION TITLE
Refresh token rotation

### DISCUSSION THREAD

**Discussion by Sourabh2001-dec** [directus/directus#5333] [Feature Requests]
Please provide an option to enable refresh token rotation in the auth

**Comment by rijkvanzanten** [comment]
What? Care to elaborate?

**Reply by Sourabh2001-dec** [reply]
Currently after the refresh token expiration reauthentication is to be performed I guess. So in refresh token rotation a new refresh token with extended expiration is issued when issuing a new access token.
Read from this article : https://auth0.com/docs/tokens/refresh-tokens/refresh-token-rotation

**Reply by rijkvanzanten** [reply]
That's what the refresh endpoint is for 👍🏻 

https://docs.directus.io/reference/api/system/authentication/#refresh

**Reply by Sourabh2001-dec** [reply]
Okay.. So in the response we get the new refresh token right??

**Reply by rijkvanzanten** [reply]
Yes, as per the docs I just linked 😉

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
